### ISSUE TITLE
FYI: NNlib.jl; depend on it?

### ISSUE THREAD

**Issue by PallHaraldsson** [sylvaticus/BetaML.jl#11]
Hi,

I just happened to (so far) only contribute activation functions to your project. Not that I use it or any of the others. I would like to help the one project where it makes the biggest impact, or one central place and this may be it:

https://github.com/FluxML/NNlib.jl/issues/224

**Comment by sylvaticus** [comment]
Just to build context, NNLIB.jl is a library of neural network primitives, like softmax and sigmoid activation function or convolutional and pooling layers.
Some things we already implemented.. some others ... hmm. I would like to keep an implementation within BetaML.... will see.... closing for now, more a TODO, but if you are willing to show how to integrate a NNLIB convolutional layer within a BetaML chain, feel free to reopen this issue or add a Notebook and make a pull request to add it to the documentation..
